__tests__/components/editor/RDFModal.test.js

```diff
@@ -0,0 +1,50 @@
+// Copyright 2018 Stanford University see Apache2.txt for license
```

Comment:
Since this file was created in 2019, maybe update the Copyright statement to 2019? Just in general, wonder if we should do a mass update on all of our code?

Reply:
¯\_(ツ)_/¯ do you want me to just change this one for now or make a ticket to do them all?